dist/openshift/cincinnati.yaml

```diff
@@ -0,0 +1,103 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: cincinnati
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: cincinnati
+    name: cincinnati
+  spec:
+    replicas: 1
+    selector:
+      app: cincinnati
+      deploymentconfig: cincinnati
+    strategy:
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: cincinnati
+          deploymentconfig: cincinnati
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          env:
+            - name: ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                    key: address
+                    name: cincinnati
+            - name: REGISTRY
+              valueFrom:
+                configMapKeyRef:
+                  key: registry
+                  name: cincinnati
+            - name: REPOSITORY
+              valueFrom:
+                configMapKeyRef:
+                  key: repository
+                  name: cincinnati
+          args: ["-vvv", "--address", "$(ADDRESS)", "--registry", "$(REGISTRY)", "--repository", "$(REPOSITORY)"]
+          imagePullPolicy: Always
+          name: cincinnati
```

Comment:
I think we should differentiate between the graph-builder and the policy-engine in the name. The two should already be able to work together so we can add the policy-engine to the template too and expose either the latter or both.

Reply:
done